PyInstaller/hooks/hook-nltk.py

```diff
@@ -0,0 +1,23 @@
+#-----------------------------------------------------------------------------
+# Copyright (c) 2005-2018, PyInstaller Development Team.
+#
+# Distributed under the terms of the GNU General Public License with exception
+# for distributing bootloader.
+#
+# The full license is in the file COPYING.txt, distributed with this software.
+#-----------------------------------------------------------------------------
+
+
+# hook for nltk
+import nltk
+from PyInstaller.utils.hooks import collect_data_files
+
+# add datas for nltk
+datas = collect_data_files('nltk', False)
+
+# loop through the data directories and add them
```

Comment:
Can you pleae be a bit more verbose here. What is the `nltk.data.path`? Why is it okay to put all data into a single directory?